nss-day-cohort-22/company-website-merciful-squids
# Gt contact page

#Steps to test
1. Pull down contact page
2. Visit localhost:8080/contact/contact.html
3. Verify everything looks good and works well
